### DISCUSSION TITLE
How to make an organization project public?

### DISCUSSION THREAD

**Discussion by quozl** [community/community#22237] [New to GitHub]
<p>We have begun to use the Projects feature of an Organization.  How do we make a Project visible to users who are not members of the organisation?</p>
<p>So far I’ve tried; Organization settings, Projects, which says “Organization project permissions … Permissions can be managed in the project’s settings on a per-project basis”, then I’ve clicked on “View all projects”, and then a project, but I cannot find permissions in settings of a project.</p>
<p>Here’s an organisation project URL; <a href="https://github.com/orgs/sugarlabs/projects/2" rel="nofollow noopener">https://github.com/orgs/sugarlabs/projects/2</a></p>
<p>Thanks!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You can find <a href="https://help.github.com/articles/changing-project-board-visibility/" rel="nofollow noopener">the instructions for changing the project board’s visibility</a> in GitHub Help.</p>
<p>Let us know if that helps!</p>

**Comment by quozl** [comment]
<p>Excellent, thanks for fixing it.</p>

**Comment by deya-0x** [comment]
<p>It seems this is no longer valid or not available for my account. I have made a Board and the “Settings” are not available in the dropdown, and I cannot currently locate another area to change a Board to public for a private repo.</p>
